### PR TITLE
PR workflow section in the CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by indeavr**
Please review if these are the rules we want to follow ! :)

### CODE REVIEW

**Review by reisr3 — CHANGES_REQUESTED** [review]
Thanks for doing this! Lemme know your thoughts

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,11 +2,30 @@
...
+
+Additionally, if you are writing a new feature, please ensure you add appropriate test cases.
+
+Follow the [Development Quick Start](#development-quick-start) to set up your local development environment.
+
+We recommend using the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) format on commit messages.
+
+### Rebasing
+
+We use the `git rebase` command to keep our commit history tidy.
```
[the remark this answers is not in the document]
Comment (deweller) [reply CONTRIBUTING.md:25]: I've been rebasing a bit where appropriate.

[the remark this answers is not in the document]
Comment (indeavr) [reply CONTRIBUTING.md:25]: Rebasing makes the history cleaner and gives better tracibility. I've seen many OS projects having this rule and I would personally prefer if we did as well. 
What do you think ? Are there any cons to it ?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,11 +2,30 @@
...
+There are many ways to contribute, including:
+1. Get involved in the protocol design decisions [Joining Discord]() !
+2. [Grab an Issue]()
 
 ## Workflow for Pull Requests
 
-🚨 Todo
+🚨 Fork; new branch; rebase; PR
+
+In order to contribute `fork` the appropriate branch `main`, create a new branch that includes your changes and open a PR to the upstream repo.
```
[the remark this answers is not in the document]
Comment (indeavr) [reply CONTRIBUTING.md:15]: Hmm, ok I agree. Its easier this way !

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,11 +2,30 @@
 
 🎈 We are so happy to have you!
 
-Todo
+We're trying to create a new way of decentralized work!
+
+There are many ways to contribute, including:
+1. Get involved in the protocol design decisions [Joining Discord]() !
+2. [Grab an Issue]()
```
Comment (deweller) [inline CONTRIBUTING.md:9]: Here's the link:

[Grab an Issue](https://github.com/engineer-dao/platform/issues)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -2,11 +2,30 @@
 
 🎈 We are so happy to have you!
 
-Todo
+We're trying to create a new way of decentralized work!
+
+There are many ways to contribute, including:
+1. Get involved in the protocol design decisions [Joining Discord]() !
```
Comment (deweller) [inline CONTRIBUTING.md:8]: Here's a link:

1. Get involved in the protocol design decisions by [joining our Discord](https://discord.gg/FGw97wHZJJ) !

**Review by deweller — DISMISSED** [review]
I added a couple of missing links in the comments.

**Pull request merged**